src/controllers/emailController.js

```diff
@@ -14,8 +13,16 @@ const sendEmail = async (req, res) => {
 
     console.log('to', to);
 
-    emailSender(to, subject, html);
-    return res.status(200).send('Email sent successfully');
+    await emailSender(to, subject, html)
+      .then(result => {
+        console.log('Email sent successfully:', result);
+        res.status(200).send(`Email sent successfully to ${to}`);
+      })
+      .catch(error => {
+        console.error('Error sending email:', error);
+        res.status(500).send('Error sending email');
+      });
+
```

Comment:
Not sure why these lines are needed since `await emailSender(to, subject, html)` is already captured by `try-catch` block.

Reply:
it helps to manage the errors in the  backend console